underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -144,7 +144,9 @@ public HdfsUnderFileSystem(AlluxioURI ufsUri, UnderFileSystemConfiguration conf,
       }
     } catch (Exception e) {
```

Comment:
Since we are at it, should we catch a more specific exception here rather than catch all? Ignoring all exception doesn't sound like a good thing to do.